build/squid/files/squid.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version='1.0'?>
+<!DOCTYPE service_bundle SYSTEM '/usr/share/lib/xml/dtd/service_bundle.dtd.1'>
+<service_bundle type='manifest' name='export'>
+  <service name='ooce/proxy/squid' type='service' version='0'>
+    <instance name='default' enabled='true'>
+      <dependency name='autofs' grouping='optional_all' restart_on='error' type='service'>
+        <service_fmri value='svc:/system/filesystem/autofs:default'/>
+      </dependency>
+      <dependency name='filesystem-local' grouping='require_all' restart_on='none' type='service'>
+        <service_fmri value='svc:/system/filesystem/local:default'/>
+      </dependency>
+      <dependency name='network' grouping='require_all' restart_on='error' type='service'>
+        <service_fmri value='svc:/milestone/network:default'/>
+      </dependency>
+      <exec_method name='start' type='method' exec='/lib/svc/method/ooce/squid start' timeout_seconds='60'>
+      </exec_method>
+      <exec_method name='stop' type='method' exec='/lib/svc/method/ooce/squid stop' timeout_seconds='120'/>
+      <property_group name='startd' type='framework'>
+        <propval name='ignore_error' type='astring' value='core,signal'/>
+      </property_group>
+      <template>
+        <common_name>
+          <loctext xml:lang='C'>Squid WebCache</loctext>
+        </common_name>
+        <documentation>
+          <doc_link name='squid-cache.org' uri='http://www.squid-cache.org'/>
+          <manpage title='squid' section='8' manpath='/usr/share/man'/>
+        </documentation>
+      </template>
+    </instance>
+    <stability value='Evolving'/>
+  </service>
+</service_bundle>
+
```

Comment:
can please you run `./tools/tidy-manifests` on the manifest file so it'll be formatted consistently with the other manifests